lms/static/sass/multicourse/_courses.scss

```diff
@@ -90,4 +121,248 @@
+      color: $m-blue-d1;
+    }
+
+    .flt-right {
+      @include float(right);
+    }
+  }
+
+
+
+  .search-facets{
+    /* rtl and ltr mixins are included to switch default layer direction of this block */
+    @include rtl() { $layout-direction: "LTR"; }
+    @include ltr() { $layout-direction: "RTL"; }
+    @include fill-parent();
+    @include omega();
+    @include box-sizing(border-box);
+    @extend %ui-depth1;
+    position: relative;
+    margin: ($baseline*2) 0 ($baseline*3.5) 0;
+    box-shadow: 1px 2px 5px $black-t0;
+    border-top: 1px solid $black;
+    border-bottom: 2px solid $black;
+    background-color: $white;
+    max-height: ($baseline*100);
+
+    @include media($bp-medium) {
+      @include span-columns(2);
+    }
+
+    @include media($bp-large) {
+      @include span-columns(3);
+    }
+
+    @include media($bp-huge) {
+      @include span-columns(3);
+    }
+
+    &.phone-menu {
+      border: medium none;
+      padding: 0;
+      overflow: visible;
+    }
+
+    &:before {
+      @include right(0);
+      position: absolute;
+      top: (-$baseline*0.15);
+      opacity: 0;
+      background-color: $white;
+      padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
+      width: ($baseline*2.5);
+      height: ($baseline/4);
+      content: "";
+    }
+
+    h2,
+    section {
+      @extend %t-icon5;
+      @extend %t-strong;
+      margin: 0 ($baseline/2);
+      border: medium none;
+      padding: ($baseline/2);
+      color: $facet-text-color;
+      font-family: $sans-serif;
+      text-transform: none;
+    }
+
+    h3 {
+      @extend %t-icon6;
+      @extend %t-strong;
+      margin: 0 ($baseline/2) ($baseline/2) ($baseline/2);
+      color: $facet-text-color;
+      font-family: $sans-serif;
+    }
+
+    section {
+      margin: 0;
+      border-top: 1px solid $courseware-button-border-color;
+      padding: ($baseline/2) 0;
+    }
+
+    a {
+      @include float(left);
+      @include box-sizing(border-box);
+      @include line-height(18.92);
+      @include transition(all $tmg-f2 ease-out 0s);
+      opacity: 1;
+      padding: 0 ($baseline*0.6);
+      width: 100%;
+      overflow: hidden;
+      text-decoration: none;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      color: $facet-text-color;
+
+      //STATE: hover, visited
+      &:hover,
+      &:visited {
+        color: $facet-text-color;
+        text-decoration: none;
+      }
+    }
+
+    ul {
+      @include padding-left(0);
+      margin: 0;
+      overflow: hidden;
+      list-style: outside none none;
+
+      &.collapse {
+        max-height: ($baseline*14);
+      }
+
+      li {
+        @include clearfix();
+        @include line-height(18.92);
+        @extend %t-icon6;
+        position: relative;
+        clear: both;
+        border-top: 1px solid $white;
+        padding: 0;
+        height: ($baseline*1.5);
+        overflow: hidden;
+
+        .count {
+          @include right($baseline*0.6);
```

Comment:
I think that we need a `float: right;` here - the `right` position specification appears to not really do anything.